Dima/netflix-roulette/src/redux/Sagas/requestSaga.ts

```diff
@@ -16,7 +22,25 @@ export const getCurrentFilm = async(id: string): Promise<IFilm> => {
   return currentFilm;
 }
 
-function* requestFilmSaga(action: IFilmsRequested) {
+interface IFilmAction {
```

Comment:
don't like that you had to create an interface just to use it in saga.  Seems redundant to me. My proposal is to use this instead:
ReturnType<typeof  filmsRequested>